scheduler/match.go

```diff
@@ -47,10 +52,54 @@ func MemoryAvailable(v float64) ogle.Matcher {
 	return &resourceMatcher{"mem", v}
 }
 
+func (m *attributeMatcher) Matches(o interface{}) (err error) {
+	offer := o.(*mesos.Offer)
+	matched := int(0)
+
+	for _, constraint := range m.SlaveConstraints {
```

Comment:
Was thinking we should have `AttributeMatch` (L93) create an AllOf matcher. but, yes, it just moving the loop around. I'm fine leaving it as is for now and I can do that refactor later.